yt_astro_analysis/radmc3d_export/RadMC3DInterface.py

```diff
@@ -364,8 +364,7 @@ def write_source_files(self, sources, wavelengths):
             fhandle.write(str(source.radius) + " ")
             fhandle.write(str(source.mass) + " ")
             fhandle.write(
-                "%f %f %f"
-                % (source.position[0], source.position[1], source.position[2])
+                f"{source.position[0]:f} {source.position[1]:f} {source.position[2]:f}"
```

Comment:
I suspect this is a breaking change too and that this function isn't fully tested

Reply:
checked. It's not breaking because `RadMC3DSource.position` is a list of floats, without explicit units.